docs/running-locally.md

```diff
@@ -199,6 +199,7 @@ Before you commit code and raise a PR, always run:
 
 ```bash
 make pre-commit -B
+CI=true make test
```

Comment:
This is wrong? CI should be false/unset unless on CI.

Reply:
Agreed, just try to skip the test. 

So, updated to skip it directly as you mentioned in https://github.com/argoproj/argo-workflows/issues/8953.